com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8.shader

```diff
@@ -51,6 +51,7 @@ Shader "Universal Render Pipeline/Nature/SpeedTree8"
             #pragma vertex SpeedTree8Vert
             #pragma fragment SpeedTree8Frag
 
+            #pragma multi_compile _ LIGHTMAP_ON
```

Comment:
It doesn't make much sense to lightmap trees, because they have a huge surface area, complicated unwrap with lots of charts so each tree will eat an entire lightmap atlas to get decent looking output. Light probes are a much better fit.

Reply:
> Before merging this, we should have a conversation around the goal of this PR. Is it some kind of GI support for trees or GI via light probes AND _lightmapping_? Please reconsider if lightmap support is needed at all, as it doesn't have much practical usage because of the huge surface area of most tree assets. Light probes (SH L2) is a much better fit.
> If we do want to support lightmaps, this needs investigation: some of the template images make me think that the UV stream is getting corrupted because it is both used to carry Speedtree data and lightmap UV data. This is done by following the data back to the speedtree importer and verifying that the data isn't overwritten/trashed by the lightmapping. gfx tests are not great for spotting problems like this.

I'm definitely not intimately knowledgeable of how lighting team have integrated probes and light-mapping, and potential implications for corrupted uv streams. It's perfectly reasonable for you guys to make the call to remove light mapping if that's the appropriate solution based on your knowledge of the functionality and performance implications versus probes etc. Confirm to remove?